alodataset/crowd_human_dataset.py

```diff
@@ -36,7 +36,23 @@ def __init__(
             return
         else:
             assert img_folder is not None, "When sample = False, img_folder must be given."
-            assert ann_file is not None, "When sample = False, ann_file must be given."
+            assert ann_file is not None or "test" in img_folder, "When sample = False and the test split is not used, ann_file must be given."
+
+        if "test" in img_folder:
+            self.dataset_dir="/data/crowdhuman/"
+            self._img_folder = img_folder
+            if isinstance(img_folder, list):
```

Comment:
- I think you'll never get here, test with `img_folder=["CrowdHuman_test", "CrowdHuman_test"]` the condition in line 41 is not satisfied.
- I don't see why it can be a list, we only have one dir of test images, don't we ?